fuzz/README.md

```diff
@@ -20,6 +20,29 @@ On Nix, you can obtain these libraries by running
 
 and then run fuzz.sh as above.
 
+# Fuzzing with weak cryptography
+
+You may wish to replace the hashing and signing code with broken crypto,
+which will be faster and enable the fuzzer to do otherwise impossible
+things such as forging signatures or finding preimages to hashes.
+
+Doing so may result in spurious bug reports since the broken crypto does
+not respect the encoding or algebraic invariants as the real crypto. We
+would like to improve this but it's a nontrivial problem -- though not
+beyond the abilities of a motivated student with a few months of time.
+Please let us know if you are interested in taking this on!
+
+Meanwhile, to use the broken crypto, simply compile (and run the fuzzing
+scripts) with
+
+    RUSTFLAGS="--cfg=bitcoin_hashes_fuzz --cfg=secp256k1_fuzz"
```

Comment:
This line is wrong mate, https://github.com/rust-bitcoin/rust-secp256k1/pull/608/files uses `sec256k1_fuzzing`, we should pick either `_fuzz` or `_fuzzing`.

Reply:
Heh, oops :) I'll change the other PR to use `_fuzz`.